test/Sentry.AspNetCore.Tests/SentryMiddlewareTests.cs

```diff
@@ -479,5 +481,37 @@ public void PopulateScope_Sdk_ContainNameAndVersion()
             Assert.Equal(Constants.SdkName, scope.Sdk.Name);
             Assert.Equal(SentryMiddleware.NameAndVersion.Version, scope.Sdk.Version);
         }
+
+        [Fact]
+        public async Task InvokeAsync_DefaultOptions_CanNotFlush()
+        {
+            var sut = _fixture.GetSut();
+            await sut.InvokeAsync(_fixture.HttpContext);
+            
+            _fixture.Options.DiagnosticLogger.Log(SentryLevel.Debug, "No events to flush.");            
+        }
+
+        [Fact]
+        public async Task InvokeAsync_FlushOnCompletedRequestWhenFalse_CanNotFlush()
+        {
+            var sut = _fixture.GetSut(); 
+            _fixture.Options.FlushOnCompletedRequest = false;
+
+            await sut.InvokeAsync(_fixture.HttpContext);
+
+            _fixture.Options.DiagnosticLogger.Log(SentryLevel.Debug, "No events to flush.");
```

Comment:
The code is not compiling in CI because you didn't add the `using` directive at the top of the file for `SentryLevel`.